server/user/user.controller.js

```diff
@@ -38,7 +38,7 @@ function create(req, res) {
   return User.restoreOrBuild(pick(body, inputAttrs))
     .then(([result]) => {
       if (result.isRejected()) return createError(CONFLICT);
-      return User.invite(result.value(), { origin });
+      return User.invite(result.value(), { origin: origin() });
```

Comment:
Well...
I would argue that better approach is changing middleware itself:
https://github.com/ExtensionEngine/boutique/blob/693e85ce0c7bcf159dc501a8e65946aaf1e1348f/server/common/origin.js#L15-L18
to attach property with getter instead of lambda...